app/controllers/assessments_controller.rb

```diff
@@ -1,4 +1,4 @@
-class AssessmentsController < ApplicationController
+class AssessmentsController < NeedsTableController
```

Comment:
no, it inherits too much stuff from NeedsTableController, what do you need to inherit exactly?

Reply:
I need `assigned_to_me` from `NeedsTableController`

Reply:
in this case it's better to copy it as is

Reply:
otherwise you can add it to the AssigningConcern as a helper method and include AssigningConcern in the AssessmentsController

Reply:
Ready to go now? https://github.com/wearefuturegov/beacon/pull/320/commits/c1efb359c05febc90c997562d61cd43782f9bb00